Modelica_DeviceDrivers/EmbeddedTargets/STM32F4/Functions/Digital/read.mo

```diff
@@ -0,0 +1,9 @@
+within Modelica_DeviceDrivers.EmbeddedTargets.STM32F4.Functions.Digital;
+impure function read
```

Comment:
impure is a Modelica 3.3 feature and should be replaced by the Impure annotation to stay consistent with the rest of the library.